app/src/main/java/org/fossasia/susi/ai/helper/Utils.kt

```diff
@@ -0,0 +1,23 @@
+package org.fossasia.susi.ai.helper
+
+import android.widget.ImageView
+import com.squareup.picasso.Picasso
+import org.fossasia.susi.ai.R
+import org.fossasia.susi.ai.rest.responses.susi.SkillData
+
+object Utils {
+    fun setSkillsImage(skillData: SkillData, imageView: ImageView) {
+        Picasso.with(imageView.context)
+                .load(getImageLink(skillData))
+                .error(R.drawable.ic_susi)
+                .fit()
+                .centerCrop()
+                .into(imageView)
+    }
+
+    fun getImageLink(skillData: SkillData): String {
+        return "BaseUrl.SUSI_DEFAULT_BASE_URL + \"/cms/getImage.png?\"" + "model=" + "$skillData.model" + "&language=" + "$skillData.language" + "&group=" +
```

Comment:
I don't know what you changed but this is still neither StringBuilder, nor kotlin string interpolation

You add changes and reviews get dismissed and changes are still not addressed

Please comment here what you are going to change so that we are on the same page

Reply:
@iamareebjamal i did not change anything, only the branch was made even and I don't know how the changes got outdated, it is strange also I did not request review recently as I did not make any change here, 

further instead of concatenating I will simply make a string and append the skillData.params as ${skillData.params} in the string , but what I don't know is how to handle the part where the space is to be replaced by "%20"

Reply:
First of all, you have to remove all the `+` signs

This is string concatenation, you should not concatenate strings, read about this

And kotlin string interpolation is not about converting

`"OK" + skillData.model + "fine:" + skillData.done`

into this:

`"OK" + "$skillData.model" + "fine:" + "$skillData.done"`

How is this better?

This is still string concatenation and you are now creating even more strings by converting `skillData.param` to a String

This should be like this

`"OK${skillData.model}fine:${skillData.done}"`

It seems you didn't read about it. Also about "converting spaces to %20", it's called encoding URLs, read about it and follow up here before sending a commit

Reply:
@iamareebjamal I have made this  : 
return "{$imageLink}model={$skillData.model}&language={$skillData.language}&group=" +
                StringBuffer(skillData.group.replace(" ", "%20")).toString() + "&image={$skillData.image}"

Now I don't understand how to replace StringBuilder